templates/terraform/examples/base_configs/test_file.go.erb

```diff
@@ -14,9 +14,9 @@ import (
 	# {Compute}{Address}_{addressBasic}
 	test_slug = "#{product}#{resource_name}_#{example.name.camelize}Example"
 	ignore_read = data[:object].all_user_properties
-														 .select(&:ignore_read)
-														 .map { |p| p.name.underscore }
-														 .concat(example.ignore_read_extras)
+							   .select(&:ignore_read)
```

Comment:
ah sorry I wasn't clear- there shouldn't be _any_ tabs in the erb files (see https://github.com/GoogleCloudPlatform/magic-modules/pull/397)